_data/sidebars/scos_dev_sidebar.yml

```diff
@@ -2953,6 +2953,10 @@ entries:
       - "202108.0"
       - "202204.0"
       - "202212.0"
+    - title: Marketplace data import
+      url: /docs/scos/dev/data-import/marketplace-data-import.html
```

Comment:
Is it intentional to add marketplace data import doc to scos?

Reply:
yes